README.md

```diff
@@ -220,19 +220,19 @@ cal.events(); // --> [ICalEvent]
 ```
 
 
-#### save(**_String_ file**[, _Function_ cb])
+#### save(**_String_ file**, _Function_ cb)
```

Comment:
Optional callback for `fs.writefile()` was deprecated in Node `v7.0.0` and errors in `v10.0.0`.
See the history section in [fs.writeFile's docs](https://nodejs.org/api/fs.html#fs_fs_writefile_file_data_options_callback)

Reply:
Oops, that totally passed me by. Thank you very much. 🙈